src/fragments/lib/graphqlapi/flutter/subscribe-data.mdx

```diff
@@ -8,15 +8,9 @@ To listen to create mutations, you can use the following code sample:
 
 ```dart
 Future<void> subscribe() async {
-  final graphQLDocument = '''subscription MySubscription {
-    onCreateBlog {
-      id
-      name
-      createdAt
-    }
-  }''';
-  final Stream<GraphQLResponse<String>> operation = Amplify.API.subscribe(
-    GraphQLRequest<String>(document: graphQLDocument),
+  final subscriptionRequest = ModelSubscriptions.onCreate(Todo.classType);
+  final Stream<GraphQLResponse<Todo>> operation = Amplify.API.subscribe(
+    subscriptionRequest,
     onEstablished: () => print('Subscription established'),
```

Comment:
Are there other functions or callbacks that can be used here? Like OnUpdate or something like that?

Reply:
Yes, there is also `onUpdate` and `onDelete`, both having same signature. Are suggesting adding those, as well?

Reply:
added